pkg/runtime/runtime.go

```diff
@@ -1208,6 +1232,18 @@ func (a *DaprRuntime) initConfiguration(s components_v1alpha1.Component) error {
 	return nil
 }
 
+func (a *DaprRuntime) shutdownConfiguration(name string, store configuration.Store) error {
```

Comment:
Is there any opportunity to move shutdown* (new functions) to a new file within the same `runtime` package?
With current state of 2K+ sloc is single file is overwhelming and it gets little tricky to review/write code.

Reply:
Agreed, and AFAIK this is already being tracked. I **can** move the shutdown functions simply to a new file, but I would propose (1) firstly understanding if it's the best way to split code - one way is to group similar functions, another will be extracting out modules. (2) separate that work in another PR

Reply:
I think we would need to weigh-in pros and cons of what approach we take to modularize code - simply taking out parts to different corresponding files or using Generics or some other way. Also, we would need to ensure that everything is good regression-wise after these changes. Given the time we have now for 1.7 release, separating out work in another PR for modularizing sounds like a better option.